common/src/main/java/org/apache/drill/common/types/Types.java

```diff
@@ -131,6 +132,17 @@ public static boolean isIntervalType(MinorType type) {
     }
   }
 
+  /**
+   * Returns true if all specified types are decimal data types.
+   *
+   * @param types types to check
+   * @return true if all specified types are decimal data type.
+   */
+  public static boolean areDecimalTypes(MajorType... types) {
```

Comment:
General enough to define as a method? Seems we'd end up having similar methods for many predicates.